charts/polkadot-base-services/templates/issuer.yaml

```diff
@@ -1,6 +1,6 @@
 {{ if .Values.createCertIssuer -}}
 apiVersion: cert-manager.io/v1alpha2
-kind: Issuer
+kind: ClusterIssuer
 metadata:
   name: letsencrypt
 spec:
```

Comment:
on line 12: being this a no namespace object, should I define somehow that the letsencrypt secret will be in the cert-manager namespaces ?

Reply:
no need for that (and it's not even possible afaik), cert-manager controller will try to find the secret in the same namespace it lives